db_stress_tool/no_batched_ops_stress.cc

```diff
@@ -290,24 +290,31 @@ class NonBatchedOpsStressTest : public StressTest {
 #endif
     }
 
-    for (const auto& s : statuses) {
-      if (s.ok()) {
-#ifndef NDEBUG
-        if (fault_fs_guard && error_count && !SharedState::filter_read_error) {
-          // Grab mutex so multiple thread don't try to print the
-          // stack trace at the same time
-          MutexLock l(thread->shared->GetMutex());
-          fprintf(stderr, "Didn't get expected error from MultiGet\n");
-          fprintf(stderr, "Callstack that injected the error\n");
-          fault_fs_guard->PrintFaultBacktrace();
-          std::terminate();
-        } else {
-#endif // NDEBUG
-          // found case
-          thread->stats.AddGets(1, 1);
 #ifndef NDEBUG
+    if (fault_fs_guard && error_count && !SharedState::filter_read_error) {
+      int stat_nok = 0;
+      for (const auto& s : statuses) {
+        if (!s.ok() && !s.IsNotFound()) {
+          stat_nok++;
         }
+      }
+
+      if (stat_nok < error_count) {
+        // Grab mutex so multiple thread don't try to print the
+        // stack trace at the same time
+        MutexLock l(thread->shared->GetMutex());
+        fprintf(stderr, "Didn't get expected error from MultiGet\n");
+        fprintf(stderr, "Callstack that injected the error\n");
```

Comment:
These two `error` keywords will be captured by `db_crashtest.py` by two errors, is it ok?

Reply:
Let me change the text so there's only one error.